src/components/app/flyouts/settings/settings.js

```diff
@@ -72,6 +83,14 @@ export class Settings extends LinkedComponent {
     this.props.toggleSimulationStatus(etag, value);
   }
 
+  toggleCheckbox = () => {
```

Comment:
This name is too generic. Name this specifically for the diagnostics opt out.

Reply:
renamed